### ISSUE TITLE
Documentation error for header with no shadow 

### ISSUE THREAD

**Issue by darkyen** [google/material-design-lite#1707]
In the layout.scss file 
```
.mdl-layout__header--seamed{
   box-shadow: none;
}
```

seems to be defined to create a header without a shadow, but in documentation the same is mentioned with `mdl-layout__header--transparent` which is mentioned twice.

Also the line 
```
.mdl-layout__header--transparent.mdl-layout__header--transparent {
  background-color: transparent;
  box-shadow: none;
}
```
Is an error right ?

**Comment by Garbee** [comment]
Double class is expected it increases specificity. And seamed class modifier has been updated, simply waiting for a site refresh.

**Comment by darkyen** [comment]
@Garbee https://github.com/google/material-design-lite/blob/master/src/layout/README.md still has double --transparent instead of --seamed in the docs. Which is why I opened the issue.

**Comment by surma** [comment]
@darkyen #1703 has been merged into `mdl-1.0` branch, which is merged into master periodically. So it will show up in master soon™
